nightly-playground/lib/nightly-playground-stack.ts

```diff
@@ -77,5 +75,6 @@ export class NightlyPlaygroundStack {
     this.stacks.push(infraStack);
 
     infraStack.addDependency(networkStack);
+    infraStack.addDependency(commonToolsStack);
```

Comment:
What is this?

Reply:
This means commonsStack needs to be ready before infra stack is deployed